ts/features/itwallet/common/utils/itwAttestationUtils.ts

```diff
@@ -0,0 +1,58 @@
+import { generate } from "@pagopa/io-react-native-crypto";
+import {
+  IntegrityContext,
+  WalletInstance,
+  WalletInstanceAttestation,
+  createCryptoContextFor
+} from "@pagopa/io-react-native-wallet";
+import uuid from "react-native-uuid";
+import { itwWalletProviderBaseUrl } from "../../../../config";
+import {
+  ensureIntegrityServiceIsReady,
+  generateIntegrityHardwareKeyTag,
+  getIntegrityContext
+} from "./itwIntegrityUtils";
+
+/**
+ * Getter for the integrity hardware keytag to be used for an {@link IntegrityContext}.
+ * @return the integrity hardware keytag to be persisted
+ */
+export const getIntegrityHardwareKeyTag = async (): Promise<string> => {
+  await ensureIntegrityServiceIsReady();
+  return await generateIntegrityHardwareKeyTag();
+};
+
+/**
+ * Register a new wallet instance with hardwareKeyTag.
+ * @param hardwareKeyTag - the hardware key tag of the integrity Context
+ */
+export const registerWalletInstance = async (hardwareKeyTag: string) => {
+  const integrityContext = getIntegrityContext(hardwareKeyTag);
+  // Check if the wallet instance has been revoked
+  await WalletInstance.createWalletInstance({
+    integrityContext,
+    walletProviderBaseUrl: itwWalletProviderBaseUrl
+  });
```

Comment:
We have to pass the `appFetch` of the app which includes our custom headers, otherwise this won't work in production. Also, we have to manage the LV 401 status code.

Reply:
Addressed in https://github.com/pagopa/io-app/pull/5861/commits/1c430babcca253fd92e71eccf631d0f2e2686522

Since we are not in a saga we have to implement custom logic to handle the 401. I think we should address it in a dedicated task.

Reply:
Can we create the task at the top of the backlog since this might be a blocking issue? Thanks.